### PR TITLE
issue #13 - enable travis and automation

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Comment by prb112** [comment]
👎 <b>FAIL</b> - Tested using <i>openjdk11</i><br><hr>
```Please refer to build log. ```

**Comment by prb112** [comment]
👍 <b>PASS</b>  - Tested using <i>openjdk11</i>

File: fhir-bulkimportexport-webapp/pom.xml [hunk 1]
```diff
@@ -116,7 +116,7 @@
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <classifier>jar</classifier>
+                            <classifier>client</classifier>
```
Comment (lmsurpre) [inline fhir-bulkimportexport-webapp/pom.xml]: what is this for?

File: fhir-parent/pom.xml [hunk 2]
```diff
@@ -341,7 +341,12 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-war-plugin</artifactId>
-                    <version>2.6</version>
+                    <version>3.2.3</version>
```
[the remark this answers is not in the document]
Comment (lmsurpre) [reply fhir-parent/pom.xml]: actually, looks like that was the `maven-jar-plugin`, not the `maven-war-plugin`, so I think we're ok

File: fhir-bulkimportexport-webapp/pom.xml (hunk 1 above)
Comment (prb112) [reply fhir-bulkimportexport-webapp/pom.xml]: The classifier enables a jar to be generated so I can pickup the jar and put it in bulkdata-operation. 
turns out the jar on the left just appends -jar.jar in the example case on maven-jar-plugin they use classifier client which generates -client.jar

File: fhir-parent/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (prb112) [reply fhir-parent/pom.xml]: I think I can work around the issue with a dependency on the plugin

**Comment by prb112** [comment]
Increased the timeout by 5 seconds

File: build/push-javadoc-to-gh-pages.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" ]; then
+
+  git clone --quiet --branch=gh-pages https://${GITHUB_OAUTH_TOKEN}@github.com/IBM/java-sdk-core.git gh-pages > /dev/null
```
Comment (lmsurpre) [inline build/push-javadoc-to-gh-pages.sh]: this should be updated to our repo, right?

Comment (prb112) [reply build/push-javadoc-to-gh-pages.sh]: done - will be reflected in the tag update

**Pull request merged**